app/src/main/java/org/thoughtcrime/securesms/permissions/RationaleDialog.kt

```diff
@@ -45,6 +47,12 @@ object RationaleDialog {
             }.also(header::addView)
         }
 
+        drawables.dropLast(1).forEach {
```

Comment:
maybe this should be behind a limit, like if there's only 3 in the it'll be `forEach` but if there's more than 3 conditionally do this "with +" logic?

Reply:
> do this "with +" logic?

I don't understand.

Reply:
nvm it's probably fine :smile: